### PR TITLE
Create NobleController.java and fix TurnController.java

### LINKED ISSUE #110: TurnController does not import needed class `Model`
Add this line to the top of the file `TurnController.java`, because it's not compiling without it.
```java
import com.example.splendormobilegame.model.Model;
```

### LINKED ISSUE #113: Implement NobleController to Handle `NOBLE_RECEIVED_ANNOUNCEMENT` Messages
The current Android client is missing functionality to handle the `NOBLE_RECEIVED_ANNOUNCEMENT` message that might be sent by the server at the end of each turn.

To address this gap, a new class called NobleController needs to be implemented. The NobleController will be responsible for processing the `NOBLE_RECEIVED_ANNOUNCEMENT` messages received from the server. Upon receiving such a message, the NobleController should update the model to reflect the acquired noble. 

Currently, there is no class like NobleController. Please, create it. Propably you should copy another controller and adjust it.

### PR DESCRIPTION
**Pull request by EaZZyHK**
I've created NobleController. I am unsure if it's appropriately made, so please check it thoroughly.

### CODE REVIEW

File: app/src/main/java/com/example/splendormobilegame/activities/GameActivity/NobleController.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    public NobleController(CustomAppCompatActivity activity) {
+        super(activity);
+        this.nobleReceivedResponse = new NobleReceivedMessageHandeler();
+    }
+
+    public NobleReceivedMessageHandeler getNobleAcquisitionResponse() {
+        return nobleReceivedResponse;
+    }
+
+    public class NobleReceivedMessageHandeler extends UserReaction {
```
Comment (kestivvi) [inline app/src/main/java/com/example/splendormobilegame/activities/GameActivity/NobleController.java]: Typo

**Review by kestivvi — CHANGES_REQUESTED** [review]
Great addition!

Please, initialize NobleController in the GameActivity and assign to it appropriate message type that it should receive.

**Comment by EaZZyHK** [comment]
I believe, that this new commit is what you have asked for

File: app/src/main/java/com/example/splendormobilegame/activities/GameActivity/GameActivity.java [hunk 2]
```diff
@@ -157,6 +158,11 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 ServerMessageType.NEW_ROOM_OWNER,
                 this.newRoomOwnerController.getNewRoomOwnerResponse()
         );
+
+        CustomWebSocketClient.getInstance().assignReactionToMessageType(
+                ServerMessageType.NOBLE_RECEIVED_ANNOUNCEMENT,
+                this.nobleController.getNobleReceivedResponse()
+        );
```
Comment (kestivvi) [inline app/src/main/java/com/example/splendormobilegame/activities/GameActivity/GameActivity.java:166]: Instance of `nobleController` is not created in the `onCreate` method, but it should like others controllers.

**Review by kestivvi — CHANGES_REQUESTED** [review]

**Review by kestivvi — APPROVED** [review]
Awesome 🎉🎉🥳

**Pull request merged**